pkg/commands/compute/setup/kv_store.go

```diff
@@ -60,11 +63,16 @@ func (o *KVStores) Configure() error {
 		var items []KVStoreItem
 
 		for key, item := range settings.Items {
+			promptMessage := "Value"
 			dv := "example"
 			if item.Value != "" {
 				dv = item.Value
 			}
-			prompt := text.BoldYellow(fmt.Sprintf("Value: [%s] ", dv))
+			if item.File != "" {
+				promptMessage = "File"
+				dv = item.File
+			}
```

Comment:
What happens when somebody has value and file? Should it not be one or the other?

Reply:
I'll fix this by checking both fields and returning an error if both are set.